### PR TITLE
Adds some types to src/runtime/debug and src/runtime/keymgmt

### PR DESCRIPTION
**Pull request by csilvestrini**

### CODE REVIEW

**Comment by csilvestrini** [comment]
Improves type coverage by a whopping 0.27 percentage points! Worth 0.27 points.

File: src/runtime/debug/arc-debug-handler.ts [hunk 1]
```diff
@@ -16,6 +16,8 @@ import {Particle} from '../recipe/particle.js';
 import {ArcStoresFetcher} from './arc-stores-fetcher.js';
 import {DevtoolsConnection} from './devtools-connection.js';
 import {enableTracingAdapter} from './tracing-adapter.js';
+import { Slot } from '../recipe/slot.js';
+import { stringify } from 'querystring';
```
Comment (mykmartin) [inline src/runtime/debug/arc-debug-handler.ts]: Nit: please remove the inner spaces; seems most auto-format tools want to add them but the vast majority of our code base doesn't have them :-/

File: src/runtime/debug/outer-port-attachment.ts [hunk 2]
```diff
@@ -8,25 +8,30 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {mapStackTrace} from '../../platform/sourcemapped-stacktrace-web.js';
+import { mapStackTrace } from '../../platform/sourcemapped-stacktrace-web.js';
+import { DevtoolsChannel } from '../../platform/devtools-channel-web.js';
+import { Arc } from '../arc.js';
```
Comment (mykmartin) [inline src/runtime/debug/outer-port-attachment.ts]: Ditto

File: src/runtime/debug/outer-port-attachment.ts [hunk 3]
```diff
@@ -8,25 +8,30 @@
...
     this.arcDevtoolsChannel.send({
       messageType: 'PecLog',
       messageBody: {name, pecMsgBody, pecMsgCount, timestamp: Date.now(), stack},
     });
   }
 
-  _extractStackFrames(stackString) {
-    const stack = [];
+  _extractStackFrames(stackString: string): StackFrame[] {
+    const stack = <StackFrame[]>[];
```
Comment (mykmartin) [inline src/runtime/debug/outer-port-attachment.ts]: const stack: StackFrame[] = [];

File: src/runtime/keymgmt/base64.ts [hunk 4]
```diff
@@ -39,10 +39,10 @@ for (let i = 0; i < 64; ++i) {
 
 export function encode(bytes: ArrayBuffer): string {
     const array = new Uint8Array(bytes);
-    const base64 = [];
+    const base64 = <string[]>[];
```
Comment (mykmartin) [inline src/runtime/keymgmt/base64.ts]: const base64: string[] = [];

File: src/runtime/keymgmt/webcrypto.ts [hunk 5]
```diff
@@ -121,21 +121,20 @@ class WebCryptoPrivateKey extends WebCryptoStorableKey<CryptoKey> implements Pri
...
 
-  static digest(str): PromiseLike<string> {
+  static digest(str: string): PromiseLike<string> {
     return WebCryptoPublicKey.sha256(str);
   }
 
-  static hex(buffer): string {
-    const hexCodes = [];
+  static hex(buffer: ArrayBuffer): string {
+    const hexCodes = <string[]>[];
```
Comment (mykmartin) [inline src/runtime/keymgmt/webcrypto.ts]: const hexCodes: string[] = [];

**Pull request merged**